backend/src/__tests__/events/debug/addDebugEmitAfterEachEmit.test.ts

```diff
@@ -0,0 +1,98 @@
+
```

Comment:
Adicionar importação da interface do Lobby, para definir o tipo dos parâmetros das funções posteriormente 
```suggestion
import Lobby from '../../../interfaces/Lobby';

```